### ISSUE TITLE
"Add" option is not highlighted in the Geoportal search

### ISSUE THREAD

**Issue by Durga07** [Esri/geoportal-server#326]
While viewing the Map,when a particular search operation is done in Geoportal search tab I am able to zoom to the particular feature level but I am unable to ADD that feature to the map. Can anyone help me with this please!
![Geoportal](https://user-images.githubusercontent.com/68694781/88362168-34569000-cd99-11ea-813e-87fddc8202f0.PNG)

**Comment by mhogeweg** [comment]
Hi,

Are these items services (map/feature/WMS etc)? Can you share the url to one of these items or its full metadata as published?

Marten
Esri

On Jul 24, 2020, at 9:19 PM, Durga07 <notifications@github.com> wrote:

﻿

While viewing the Map,when a particular search operation is done in Geoportal search tab I am able to zoom to the particular feature level but I am unable to ADD that feature to the map. Can anyone help me with this please!
[Geoportal]<https://urldefense.com/v3/__https://user-images.githubusercontent.com/68694781/88362168-34569000-cd99-11ea-813e-87fddc8202f0.PNG__;!!CKZwjTOV!hv_TEqj04vSVv4ak-b8eOktca0mynYTSc_qGil_DwJGtoRQ0Bz8b62dYnlaXHQ$>

**Comment by Durga07** [comment]
I'm sharing the metadata file sir

[Files.zip](https://github.com/Esri/geoportal-server/files/4974927/Files.zip)

**Comment by mhogeweg** [comment]
hi, tank you for sharing that. It appears you have registered the feature classes in your geodatabase. This will not get the 'add to map' link as that is for web services like WMS, ArcGIS Map Services, ArcGIS Feature Services and such. Consider that the Geoportal user (like myself) will not be able to connect to your personal geodatabase.

If you have access to ArcGIS Server or ArcGIS Online, you could publish the feature class there and then you'd register the map/feature service in the geoportal. That would then be a layer that can be added to the map viewer.

If you don't have ArcGIS Online you can setup a free personal account, see: https://doc.arcgis.com/en/arcgis-online/get-started/create-account.htm#ESRI_SECTION1_D91DD2A709AE4FB68A9CC095F1688E05

**Comment by Durga07** [comment]
Hi
Can Geoportal be installed on Windows 10 or should it be done on Windows Server?

**Comment by mhogeweg** [comment]
Geoportal can run fine on windows 10 or any other operating system that can run java, tomcat, and elasticsearch

**Comment by Durga07** [comment]
Hi
I am trying to harvest the files from my PC(this is not the system on which I installed the Geoportal) to the Geoportal by using SINK as the input broker, in the Harvester-Processes section it has shown the task is completed. But these files are not harvested to the Geoportal. Will the files that are present in the local computer(on which Geoportal is installed) only be harvested through this broker?
I tried to harvest the folders from the sample services (http://sampleserver1.arcgisonline.com/ArcGIS/rest/services) to the Geoportal. These folders are also not harvested to the Geoportal.
